lib/bike_brigade_web/live/sms_message_live/form_component.html.heex

```diff
@@ -21,12 +21,16 @@
         Message
       <% end %>
       <div class="mt-1 rounded-md shadow-sm">
-        <%= textarea f, :body, required: true, phx_debounce: "blur", class: "block w-full px-3 py-2 placeholder-gray-400 transition duration-150 ease-in-out border border-gray-300 rounded-md appearance-none focus:outline-none focus:ring-blue focus:border-blue-300 sm:text-sm sm:leading-5" %>
+        <%= textarea f, :sms_message_form, required: true, phx_debounce: "blur", class: "block w-full px-3 py-2 placeholder-gray-400 transition duration-150 ease-in-out border border-gray-300 rounded-md appearance-none focus:outline-none focus:ring-blue focus:border-blue-300 sm:text-sm sm:leading-5" %>
```

Comment:
this parameter is the name of the input field, that's what we use on the backend to determine what to do with that input. I think changing it will break the entire form.